src/components/transactions/SignedMessagesHelpLink/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { Box, SvgIcon, Typography } from '@mui/material'
+import InfoIcon from '@/public/images/notifications/info.svg'
+import ExternalLink from '@/components/common/ExternalLink'
+import { useAppSelector } from '@/store'
+import { selectSafeMessages } from '@/store/safeMessagesSlice'
+
+const SignedMessagesHelpLink = () => {
+  const safeMessages = useAppSelector(selectSafeMessages)
+  const safeMessagesCount = safeMessages.data?.results.length ?? 0
+
+  if (safeMessagesCount === 0) {
+    return null
+  }
```

Comment:
Why hide it when no messages? The link is still relevant to the page.

Reply:
When we have no messages we have the link in the placeholder. The issue report also mentions it https://github.com/safe-global/web-core/issues/1430 in the `Steps to reproduce`

Reply:
Right, just saw it on your second screenshot. 👍